.github/workflows/nano-style-check.yml

```diff
@@ -0,0 +1,37 @@
+name: Nano code style checking
+
+# Controls when the action will run. 
+on:
+  push:
+    branches: [ branch-2.0 ]
+  pull_request:
+    branches: [ branch-2.0 ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  style-check:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version: [3.7]
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+      
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install dependencies
+        run: pip install pycodestyle
+      
+      # Runs a single command using the runners shell
+      - name: Style checking
+        run: bash python/nano/test/run-nano-codestyle-test.sh
+        env:
+            ANALYTICS_ZOO_ROOT: ${{ github.workspace }}
```

Comment:
Why ANALYTICS_ZOO_ROOT?

Reply:
This environment variable is used in many script. 

You mean changing to BIGDL_ROOT?

Reply:
which scripts does this one use? And yes, changing to BIGDL_ROOT unless it's too much work

Reply:
> which scripts does this one use?

**This test runs**
```
line 35   run: bash python/nano/test/run-nano-codestyle-test.sh
```

**Scripts using ANALYTICS_ZOO_ROOT:**
```
./scala/dllib/src/main/scala/com/intel/analytics/bigdl/dllib/example/run-scala-example-test-local.sh
./python/orca/dev/example/run-example-test-ray.sh
./python/orca/dev/example/run-example-test-ray-integration.sh
./python/nano/test/run-nano-test.sh
./python/nano/test/run-nano-test-ipex.sh
./python/nano/test/run-nano-type-test.sh
./python/nano/test/run-nano-codestyle-test.sh
./apps/run-scala-app-test.sh
./apps/run-app-tests-pytorch-pip.sh
./apps/run-app-tests-pip.sh
```

Reply:
> And yes, changing to BIGDL_ROOT unless it's too much work

Changing code is easy. Changing many jenkins job configs one by one could take some time. @jason-dai